charts/telepresence/templates/clientRbac/namespace-scope.yaml

```diff
@@ -0,0 +1,41 @@
+# These are the namespace-scoped rbac roles + bindings that will be used by users
+# who want to use telepresence once its components have been set
+# up in the cluster.
+{{- if .Values.clientRbac.create }}
+{{- if .Values.clientRbac.namespaced }}
+
```

Comment:
Should we also add the check here to ensure that the user has subjects set in their values.yaml?

Reply:
i don't think we need to duplicate it -- the check in `cluster-scope.yaml` [isn't conditional](https://github.com/telepresenceio/telepresence/pull/1845/files#diff-4ea4fc391ecffb9d40967c748d5dc01bfa589f6061b2786f8ad5d738bacaa9bcR7-R11) so we should hit it regardless of whether `clientRbac.namespaced` is set

Reply:
ah good point :)